src/cli/update.ts

```diff
@@ -88,7 +88,7 @@ export default function (program: RootCmd) {
           }
         }
       } catch (e) {
-        console.error(e.message);
+        console.error(`Error updating template files: ${e.message}`);
```

Comment:
```suggestion
        console.error(e, `Error updating template files`);
```

Reply:
All messages updated to display complete `e`